rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -39,13 +40,12 @@ def next(self):
         return self.__next__()
 
     def __next__(self):
-        if b.getRemainingImages(self.loader._handle) < self.bs:
-            raise StopIteration
-
-        if self.loader.run() != 0:
+        if self.loader.rocal_run() != 0:
             raise StopIteration
+        else:
+            self.output_tensor_list = self.loader.get_output_tensors()
 
-        self.loader.copyImage(self.out_image)
+        self.output_tensor_list[0].copy_data_numpy(self.out_image)
         return self.out_image , self.out_tensor
```

Comment:
self.out_tensor is assigned as None.

Remove this from return statement

Reply:
Check if this iterator is used anywhere in th examples ?

If not, we can remove this as this is being added as a part of the generic.py iterator